oam/install_scripts/mariadb-columnstore-start.sh

```diff
@@ -2,7 +2,6 @@
 
 # This script allows to gracefully start MCS
 
-/bin/systemctl start mcs-storagemanager
```

Comment:
we no longer always start service to avoid logging a failure for mcs-storagemanager when it's not needed